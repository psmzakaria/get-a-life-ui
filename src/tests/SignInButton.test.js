import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import SignInButton from "./../components/SignInButton";

it("should render the sign in button", () => {
  const renderer = new ShallowRenderer();
  renderer.render(<SignInButton />);
  const output = renderer.getRenderOutput();

  expect(output).toMatchSnapshot();
});
