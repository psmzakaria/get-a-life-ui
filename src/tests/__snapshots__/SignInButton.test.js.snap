// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the sign in button 1`] = `
<div>
  <Button
    as="button"
    color="blue"
    fluid={true}
    id="signin"
    onClick={undefined}
    role="button"
    type="submit"
  >
    Sign In
  </Button>
  <Divider />
  <div>
    <p
      className="text-align-center"
    >
      Don't have an account?
      <a
        className="cursor-pointer"
        onClick={[Function]}
      >
         
         Sign Up Here
      </a>
    </p>
  </div>
</div>
`;
